composer.json

```diff
@@ -106,7 +106,7 @@
         "league/flysystem-sftp-v3": "^3.0",
         "mockery/mockery": "^1.6",
         "nyholm/psr7": "^1.2",
-        "orchestra/testbench-core": "^9.0.15",
+        "orchestra/testbench-core": "dev-laravel-51725",
```

Comment:
Testbench was setting the `$latestResponse` property in its version of `TestCase`, but rather than just deprecate the property, @crynobone was happy to update Testbench, and will tag a proper release if this PR is accepted.